### ISSUE TITLE
Allowing users to add more distros

### ISSUE THREAD

**Issue by rghv234** [blend-os/blendOS#4]
Hey guys, this is a very cool project, one of my dream come true to be honest!

I have a question, will it allow users to "add" more distros if they wanted to?

Say if they wanted to run gentoo, alpine or void linux, alongside the distros that it already ships with

Will it be possible in the near future?

**Comment by RudraSwat** [comment]
Yep, I'll be making it easier to add more distributions in future releases. Let's use this bug to track support for other distributions.

**Comment by rghv234** [comment]
Awesome! I feel that this concept is a little more stable than bedrock linux, love the project!
